app/models/notifications/Entry.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.notifications
+
+import models.UcrBlock
+import play.api.libs.json.Json
+
+case class Entry(ucrBlock: Option[UcrBlock] = None, goodsItem: Seq[GoodsItem] = Seq.empty, entryStatus: Option[EntryStatus] = None) {
```

Comment:
What's an "Entry"? Is there a better name we can think of?

Reply:
As I remember it's element from schema.

Reply:
Fair enough, just seems a big of a general name to me, like `Thing` or `Element` etc

Reply:
This is the name given by ILE schema and I'd prefer to stay with the naming even if it's not perfect.